### PR TITLE
Simple script installation

### PR DESCRIPTION
**Pull request by Tansito**
### Summary

Related with #129 I created some simple scripts to simplify installation process. This doesn't mean that these scripts are going to disable the development of a future CLI so issue #129 and #295 are still relevant.

### Details and comments

- Created three scripts that will install a `cluster`, a `helm` or both of them at once.
- Almost all the information required to run the scripts is collected [here](https://github.com/Qiskit-Extensions/quantum-serverless/pull/520/files#diff-8cbf1add84f5f682cd9528c99e93127ade82ce32ef8a4147d54cee82d900e422R54).

### CODE REVIEW

**Comment by pacomf** [comment]
@psschwei @akihikokuroda can you test it from your side?

**Comment by psschwei** [comment]
I can check next week when I'm back from OSSC

**Review by psschwei — CHANGES_REQUESTED** [review]
Two things:

To get around the script location issue, I'd recommend moving this (and the other install scripts) to a top-level `hack/` or `scripts/` directory (either of the overall repo or within `infrastructure/`).

Also, this fails for me when running on a new cluster with the following error:
```
wait.go:66: [debug] beginning wait for 11 resources with timeout of 1m0s
Error: unable to build kubernetes objects from release manifest: [resource mapping not found for name: "loki" namespace: "" from "": no matches for kind "GrafanaAgent" in version "monitoring.grafana.com/v1alpha1"
ensure CRDs are installed first, resource mapping not found for name: "loki" namespace: "" from "": no matches for kind "LogsInstance" in version "monitoring.grafana.com/v1alpha1"
ensure CRDs are installed first, resource mapping not found for name: "loki" namespace: "" from "": no matches for kind "PodLogs" in version "monitoring.grafana.com/v1alpha1"
ensure CRDs are installed first]
helm.go:84: [debug] [resource mapping not found for name: "loki" namespace: "" from "": no matches for kind "GrafanaAgent" in version "monitoring.grafana.com/v1alpha1"
ensure CRDs are installed first, resource mapping not found for name: "loki" namespace: "" from "": no matches for kind "LogsInstance" in version "monitoring.grafana.com/v1alpha1"
ensure CRDs are installed first, resource mapping not found for name: "loki" namespace: "" from "": no matches for kind "PodLogs" in version "monitoring.grafana.com/v1alpha1"
ensure CRDs are installed first]
unable to build kubernetes objects from release manifest
helm.sh/helm/v3/pkg/action.(*Install).RunWithContext
        helm.sh/helm/v3/pkg/action/install.go:277
main.runInstall
        helm.sh/helm/v3/cmd/helm/install.go:278
main.newUpgradeCmd.func2
        helm.sh/helm/v3/cmd/helm/upgrade.go:122
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.5.0/command.go:872
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.5.0/command.go:990
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.5.0/command.go:918
main.main
        helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
        runtime/proc.go:250
runtime.goexit
        runtime/asm_amd64.s:1571
```
@pacomf @IceKhan13 could one of you try installing on a new cluster? (doesn't have to be IKS, locally on minikube/kind would work too for this purpose) just want to verify that it's not just me...

File: infrastructure/helm/quantumserverless/install.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+# Ask for the minimum information to configure the values
+echo "
+All the requirements all fulfilled. Now you are going to be asked for some information to continue:"
+
+# Looking for ingress public end-point
+if [ -z "$INGRESS_PUBLIC_END_POINT" ]
+then
+  read -p "
+  * Ingress public end-point of your cluster: " INGRESS_PUBLIC_END_POINT
```
[the remark this answers is not in the document]
Comment (pacomf) [reply infrastructure/helm/quantumserverless/install.sh:34]: good suggestion!

**Comment by pacomf** [comment]
@psschwei thank you! David will take care when he is back :)

**Comment by psschwei** [comment]
I fixed the missing CRDs issue at some point (it was also hitting us in other places), so if you rebase on main you should pull in that fix

**Comment by Tansito** [comment]
Thanks to take a look @psschwei ! This week I'm going to need to deploy it so I will take a look more in-depth. Maybe we will need to update some configurations due to the last changes 👀

**Comment by psschwei** [comment]
@Tansito just checking on this one -- is this something we're still planning on, or have some of the other infrastructure revampings superseded it?

**Comment by Tansito** [comment]
Thanks for the reminder @psschwei, no, let's close it by now. A lot of priorities opened at this moment and we can always recover it 👍

**Pull request closed without merging**